mutable/slice.go

```diff
@@ -0,0 +1,95 @@
+package mutable
```

Comment:
```go
package mutable
```

or

```
package lom
```

?

It does matters, because it will be the default package alias on import.

Reply:
No one will get lom, mutable as a package is good.

Don't enforce your import alias as package name 😅😬